webui/common/src/components/Logs/Logs.test.tsx

```diff
@@ -40,7 +40,7 @@ describe('Logs', () => {
       loading: false,
       class: '',
       errors: createAlertErrors(false),
-      upateRequestParameter: sinon.spy()
+      update: sinon.spy()
```

Comment:
I meant to change it to `updateRequestParameter`, because `update` was misspelled

Reply:
ohh woops - updated